R/add_child.R

```diff
@@ -1,13 +1,22 @@
 #' Write R Chunk to Add Child Document
 #'
-#' @param x An additional section to add into the template. Options for additional sections are in the 'skeleton' folder. Appropriate files are .qmd files and are formatted as such: XX_section.qmd (i.e., not a, b, c... subfiles).
-#' @param label Description of the child document being added. It should be short- one or two words, maximum.
+#' @param x An additional section to add into the template. Options
+#' for additional sections are in the 'skeleton' folder.
+#' Appropriate files are .qmd files and are formatted as such:
+#' XX_section.qmd (i.e., not a, b, c... subfiles).
+#' @param label Description of the child document being added.
+#' It should be short- one or two words, maximum.
 #'
-#' @return Formatting R chunk for child document to add section into the template/skeleton.
-#'         Utilize the cat() function to implement into readable text.
+#' @return Formatting R chunk for child document to add section
+#' into the template/skeleton. Utilize the cat() function to
+#' implement into readable text.
 #' @export
 #'
-#' @examples add_child("test_quarto.qmd", label = "test_doc")
+#' @examples
+#'\dontrun{
+#' add_child(x = "test_quarto.qmd",
```

Comment:
Is the function set up that you need to add the .qmd extension? I thought I made it so they didn't need that. If it's the latter, then the extension will get replicated

Reply:
Yes, I believe so, since I didn't change the function's arguments at all- only separated them onto separate lines. In `create_template.R`, `add_child()` is used in a way that also suggests that x contains .qmd and the label doesn't: https://github.com/nmfs-ost/asar/blob/a46b0a4d50545fee673444152a1edfb1dfebe6b9/R/create_template.R#L512C1-L542C10